CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+Version 0.8.1 (2022-02-01)
+---------------------------
+
+- Adds logic to calculate the CPU quota periodically.
+- Changes CLI quota command from ``disk-usage-update`` to ``resource-usage-update`` to also perform the CPU quota calculation.
+- Removes extra ``QuotaResourceType`` enum in favor of ``ResourceType.name``.
+
```

Comment:
Let's add one news item:

- Fixes quota update functions to handle exceptional situation as continuable errors.  _(describing the recent PR)_

Can you please also add a line for Vlad to `AUTHORS.rst`?